src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/output/OutputSpreadsheet.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.output
+
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
+import uk.gov.justice.digital.hmpps.pecs.jpc.calculator.MovePrice
+import java.io.File
+import java.io.FileOutputStream
+
+class OutputSpreadsheet {
+
+    fun generateSpreadsheet(prices: Sequence<MovePrice>) : File {
+        val workbook = XSSFWorkbook()
+        val sheet = workbook.createSheet("Standard Moves")
+
+        // Row for Header
+        val headerRow = sheet.createRow(0)
+        headerRow.createCell(0).setCellValue("Ref")
+        headerRow.createCell(1).setCellValue("From")
+        headerRow.createCell(2).setCellValue("To")
+        headerRow.createCell(3).setCellValue("Price")
+
+        prices.forEachIndexed{i, p ->
+            val row = sheet.createRow(i+1)
+
+            with(p.movePersonJourneysEvents.move){
+                row.createCell(0).setCellValue(reference)
+                row.createCell(1).setCellValue(fromLocation)
+                row.createCell(2).setCellValue(toLocation)
+            }
+
+            p.totalInPence()?.let{
+                row.createCell(3).setCellValue(it.toDouble()) } ?:
+                row.createCell(3).setCellValue("NOT PRESENT")
+        }
+
+        return createTempFile(suffix = "xlsx").apply {
+            val fileOut = FileOutputStream(this)
```

Comment:
So you can use the inline `use` function here to close stuff.  I think something like:

`FileOutputStream(this).use { fo -> workbook.use { wb-> wb.write(fo) } }`